tools/helper_default_based_schedule

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/perl
+
+# Tool to merge YAML_SCHEDULE_DEFAULT  with YAML_SCHEDULE_FLOWS
+# and YAML_SCH§EDULE.
+
+# So far all pointers should be provided as environment
+# variables
+
+use strict;
+use warnings;
+
+use File::Basename;
```

Comment:
do we use this somewhere?

Reply:
If I exclude File::Basename,then fileparse($default_path, '.yaml') will throw an error.